<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.apache.commons.math3.exception.util.ExceptionContextTest" tests="5" skipped="0" failures="0" errors="0" timestamp="2020-10-17T21:41:13" hostname="DYLAN-DESKTOP" time="0.01">
  <properties/>
  <testcase name="testNoArgAddMessage" classname="org.apache.commons.math3.exception.util.ExceptionContextTest" time="0.0"/>
  <testcase name="testSerializeUnserializable" classname="org.apache.commons.math3.exception.util.ExceptionContextTest" time="0.007"/>
  <testcase name="testMessageChain" classname="org.apache.commons.math3.exception.util.ExceptionContextTest" time="0.0"/>
  <testcase name="testSerialize" classname="org.apache.commons.math3.exception.util.ExceptionContextTest" time="0.002"/>
  <testcase name="testContext" classname="org.apache.commons.math3.exception.util.ExceptionContextTest" time="0.0"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
